bin/dehydrated

```diff
@@ -0,0 +1,1116 @@
+#!/usr/bin/env bash
```

Comment:
Possible, but all we need is the script from that module, and we will want to pin it to a version. So the build will either `curl -O` a url (with version) or cloning a submodule, checking out the right version there, and copying the script from there to out `bin` path.